etc/config/hlsl.defaults.properties

```diff
@@ -10,6 +10,10 @@ group.rga.compilerType=rga
 compiler.rga_default.exe=/usr/rga/rga
 compiler.rga_default.dxcPath=/usr/dxc-artifacts/bin/dxc
 
+group.clang.compilers=hlsl_clang_default
+group.rga.compilerType=clang
```

Comment:
Typo, `group.rga`

Reply:
Also should have the -x hlsl option set here too

Reply:
Thanks, fixed.